src/userSync.js

```diff
@@ -0,0 +1,192 @@
+import * as utils from 'src/utils';
+
+const userSync = exports;
+// Set user sync config default values which can be overridden by the publisher
+const userSyncDefaultConfig = {
+  syncEnabled: true,
+  pixelEnabled: true,
+  syncsPerBidder: 5
+}
+
+// A queue of user syncs for each adapter
+// Let setQueue() set the defaults
+let queue;
```

Comment:
@dbemiller and the team have been discussing how this pattern allows global mutable state because any caller for adding pixels will be stored here. As an alternative method, he suggests to use a constructor function to encapsulate the data to a single instance. I'm inclined to agree with this pattern and like how it makes it easier for us to refactor to "prebid instance" in the future if we decide to do so. 

@grevory @snapwich what do you think about this pattern? Dave put up an example here: https://github.com/prebid/Prebid.js/pull/1453 if you want to take a look at something concrete.